### PR TITLE
Clone TKG from its own repo

### PR DESCRIPTION
**Pull request by renfeiw**
- modify get.sh to separate TKG from openj9
- modify JenkinsfileBase to incorporate TKG repo
- modify multiple files to update testKitGen to TKG
- update documents

Signed-off-by: Renfei Wang <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
related PR: https://github.com/eclipse/openj9/pull/7885

File: doc/userGuide.md [hunk 1]
```diff
@@ -211,9 +211,13 @@ Usage : get.sh  --testdir|-t openjdktestdir
...
-                [--openj9_repo ] : optional. OpenJ9 git repo. Default value: https://github.com/eclipse/openj9.git is used if not provided
-                [--openj9_sha ] : optional. OpenJ9 pull request sha.
-                [--openj9_branch ] : optional. OpenJ9 branch.
+                [--clone_openj9 ] : optional. ture or false. If set to true, get.sh will download openj9 repo functional tests. Default to false'
+                [--openj9_repo ] : optional. OpenJ9 git repo. Default value https://github.com/eclipse/openj9.git is used if not provided'
+                [--openj9_sha ] : optional. OpenJ9 pull request sha.'
+                [--openj9_branch ] : optional. OpenJ9 branch.'
+                [--tkg_repo ] : optional. TKG git repo. Default value https://github.com/AdoptOpenJDK/TKG.git is used if not provided'
+                [--tkg_sha ] : optional. TkG pull request sha.'
+                [--tkg_branch ] : optional. TKG branch.'
```
Comment (llxia) [inline doc/userGuide.md]: - Default value master is used if not provided

- Please remove `'` at the end of each option.

Comment (renfeiw) [reply doc/userGuide.md]: updated

**Comment by llxia** [comment]
@renfeiw is doing final testing. We should be able to merge this soon.

**Comment by llxia** [comment]
Thanks @renfeiw 
This PR is tested in internally.

**Pull request merged**